{{ cookiecutter.project_shortname }}/{{ cookiecutter.package_name }}/ext.py

```diff
@@ -28,7 +29,6 @@ def init_app(self, app):
 
     def init_config(self, app):
         """Initialize configuration."""
-        app.config.setdefault(
-            "{{ cookiecutter.config_prefix}}_BASE_TEMPLATE",
-            app.config.get("BASE_TEMPLATE",
```

Comment:
Actually looking at the documentation in Invenio-Theme is not correct :-) It also won't do the right job….I'll open an issue